www/assets/js/components/SearchFilterDrawer.tsx

```diff
@@ -53,12 +53,16 @@ export default function SearchFilterDrawer(props: Props) {
   return drawerOpen ? (
     <div className="search-filter-drawer-open">
       <div className="controls">
-        <i className="material-icons" onClick={closeDrawer}>
-          close
-        </i>
+        <button
+          className="bg-transparent border-0"
+          onClick={closeDrawer}
+          type="button"
+        >
+          <span className="material-icons">close</span>
```

Comment:
**Minor Suggestion:** Hide the icon and label the drawer "Close Search Filters".